ui/togeather/src/.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: trusty
```

Comment:
This file should be put into the root of repository and according to travis documentation it is better to set the dist to `xenial` and remove the `sudo: false`.

Reply:
If you set it up correctly you will see travis running the jobs for this pull request already.